src/devices/bus/epson_qx/option.cpp

```diff
@@ -0,0 +1,211 @@
+// license:BSD-3-Clause
+// copyright-holders:Brian Johnson
+/*******************************************************************
+ *
+ * Epson QX-10 Expansion emulation
+ *
+ *******************************************************************/
+#include "emu.h"
+#include "option.h"
+
+DEFINE_DEVICE_TYPE(OPTION_BUS_SLOT, bus::epson_qx::option_slot_device, "option_slot", "QX-10 Option slot")
+DEFINE_DEVICE_TYPE(QX_OPTION_BUS, bus::epson_qx::option_bus_device, "qx_option_bus", "QX-10 Option Bus")
```

Comment:
Remember device “short names” exist in a single namespace shared with driver names – `option_slot` is definitely too generic and liable to cause a conflict.  probably better to have `epson_qx_` prefixes on both the names.